### PR TITLE
[zh]sync observability/distributed-tracing/telemetry-api

### PR DESCRIPTION
**Pull request by asa3311**
sync telemetry api
link #12958

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/observability/distributed-tracing/telemetry-api/index.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+owner: istio/wg-policies-and-telemetry-maintainers
+test: yes
+---
+
+Istio 提供了配置高级链路追踪选项的功能，例如采样率和向已采集的 Span 中添加自定义标签。
+本任务将向您展示如何使用 Telemetry API 自定义链路追踪选项。
+
+## 开始之前{#before-you-begin}
+
+1. 请确保您的应用程序按照[这里](/docs/tasks/observability/distributed-tracing/overview/)所描述的方式配置链路追踪头（tracing headers）。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/observability/distributed-tracing/telemetry-api/index.md:15]: use zh doc link instead.

File: content/zh/docs/tasks/observability/distributed-tracing/telemetry-api/index.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+2. 请根据你首选的追踪后端，根据[Integrations](/docs/ops/integrations/)追踪安装指南安装适当的插件，
```
Comment (kebe7jun) [inline content/zh/docs/tasks/observability/distributed-tracing/telemetry-api/index.md:17]:
```suggestion
1. 请根据您首选的追踪后端，根据[集成](/zh/docs/ops/integrations/)追踪安装指南安装适当的插件，
```

File: content/zh/docs/tasks/observability/distributed-tracing/telemetry-api/index.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+   并配置你的 Istio 代理将链路追踪信息发送到链路追踪部署服务端。
+
+## 安装{#installation}
+
+在此示例中，我们将发送跟踪信息到[`zipkin`](/docs/ops/integrations/zipkin/)，请确保已安装它：
```
Comment (kebe7jun) [inline content/zh/docs/tasks/observability/distributed-tracing/telemetry-api/index.md:22]: zh link

File: content/zh/docs/tasks/observability/distributed-tracing/telemetry-api/index.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+    - name: zipkin
+      zipkin:
+        service: zipkin.istio-system.svc.cluster.local
+        port: 9411
+        maxTagLength: <VALUE>
+{{< /text >}}
+
+## 验证结果{#verify-the-results}
+
+您可以使用[Zipkin UI](/docs/zh/tasks/observability/distributed-tracing/zipkin/)来验证结果。
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/distributed-tracing/telemetry-api/index.md:178]:
```suggestion
您可以使用[Zipkin UI](/zh/docs/tasks/observability/distributed-tracing/zipkin/)来验证结果。
```

**Review by zirain — APPROVED** [review]

**Pull request merged**